[tflite] fix calculating of output pixels

freedomtan opened the pull request:
fix an issue reported by issue #22310

freedomtan committed:
[tflite] fix calculating of output pixels

fix an issue reported by issue #22310

qlzh727 reviewed (dismissed):
Thank for the fix.

qlzh727 commented:
Can u quickly fix the clang format?

https://source.cloud.google.com/results/invocations/2f645f3a-19b6-4ef3-a028-6629e673cae8/targets/%2F%2Ftensorflow%2Ftools%2Fci_build:gen_ci_clang_format_out/log

diff --git a/tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h b/tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h
index 7e09d4bc79..21ad39a6bf 100644
--- a/tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h
+++ b/tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h
@@ -80,8 +80,7 @@ void resize(T* out, uint8_t* in, int image_height, int image_width,
   interpreter->Invoke();
   auto output = interpreter->typed_tensor<float>(2);
-  auto output_number_of_pixels =
-      wanted_height * wanted_width * wanted_channels;
+  auto output_number_of_pixels = wanted_height * wanted_width * wanted_channels;
   for (int i = 0; i < output_number_of_pixels; i++) {
     if (s->input_floating)

freedomtan committed:
fix style

> clang-format -i --style Google tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h

freedomtan commented:
@qlzh727 thanks. clang-format done
